public/app/features/search/components/ActionRow.tsx

```diff
@@ -37,8 +39,10 @@ export const ActionRow: FC<Props> = ({
 
   return (
     <div className={styles.actionRow}>
-      <HorizontalGroup spacing="md">
-        {!hideLayout ? <RadioButtonGroup options={layoutOptions} onChange={onLayoutChange} value={layout} /> : null}
+      <HorizontalGroup spacing="md" width="100%">
```

Comment:
this should not be width 100% ? will squeeze the tag select and not allow it to expan d

Reply:
hmm, maybe I've applied a wrong change during master merge.

Reply:
It actually has to be `width="auto"` for tag filter to expand properly.